demo/demo-etcd/test-client/pom.xml

```diff
@@ -45,6 +45,22 @@
     <dependency>
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>registry-local</artifactId>
+      <exclusions>
+        <exclusion>
+          <artifactId>commons-logging</artifactId>
+          <groupId>commons-logging</groupId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
registry-local may be not dependent on commons-logging. if it is exclude in registry-local, not here.